etcd-client/src/kv/cache.rs

```diff
@@ -118,11 +124,12 @@ impl Cache {
     /// Clean cache
     #[allow(unsafe_code)]
     pub async fn clean(&self) {
+        // Don't change clear sequence
```

Comment:
Why do we change the sequence here?

Reply:
This clear sequence will make sure consistency between hashtable and lru-queue.